console-subscriber/tests/wake.rs

```diff
@@ -4,35 +4,6 @@ use std::time::Duration;
 use support::{assert_task, ExpectedTask};
 use tokio::time::sleep;
 
-#[test]
```

Comment:
How about we keep the tests but just ignore them by the `#[ignore]` macro? In the future, we can add it back easily.

Reply:
I'd prefer to just remove. We can use #473 to track the need to put them back.